cmd/flow-db.go

```diff
@@ -0,0 +1,223 @@
+package cmd
+
+import (
+	"database/sql"
+	"encoding/json"
+	"os"
+	"sync"
+
+	"github.com/netobserv/flowlogs-pipeline/pkg/config"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline/utils"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline/write/grpc"
+	"github.com/netobserv/flowlogs-pipeline/pkg/pipeline/write/grpc/genericmap"
+
+	// need to import the sqlite3 driver
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/spf13/cobra"
+)
+
+var flowsDBCmd = &cobra.Command{
+	Use:   "get-flows-db",
+	Short: "",
+	Long:  "",
+	Run:   runFlowDBQuery,
+}
+
+const flowsDB = "/tmp/flows.db"
+
+func runFlowDBQuery(_ *cobra.Command, _ []string) {
+	wg := sync.WaitGroup{}
+	wg.Add(len(ports))
+	for i := range ports {
+		go func(idx int) {
+			defer wg.Done()
+			queryFlowDB(ports[idx])
+		}(i)
+	}
+	wg.Wait()
+}
+
+func queryFlowDB(port int) {
+	os.Remove(flowsDB) // I delete the file to avoid duplicated records.
```

Comment:
In the other commands, I use the following code to allow custom filename or fallback on date / time:
```
	if len(filename) > 0 {
		log.Infof("Starting Flow Capture for %s...", filename)
	} else {
		log.Infof("Starting Flow Capture...")
		filename = strings.Replace(
			time.Now().UTC().Format(time.RFC3339),
			":", "", -1) // get rid of offensive colons
	}
```

We should make a util and have the same logic everywhere

Reply:
no needed now since the commands are merged

Reply:
It's still needed to get both json + db files as:
- `2024-03-27T112904Z.json`
- `2024-03-27T112904Z.db`

Currently the db is always `flows.db`. I can make the change to pass `filename` arg from `runFlowCaptureOnAddr` if you want